### PR TITLE
Create Infrastructure to test JS on CI

### PR DESCRIPTION
**Pull request by USAMAWIZARD**
Test JS test cases on CI.
https://github.com/ant-media/Ant-Media-Server/issues/4140

### CODE REVIEW

**Review by mustafaboleken — CHANGES_REQUESTED** [review]
Instead of community edition, please use enterprise edition.

**Review by mustafaboleken — APPROVED** [review]
lgtm

File: .github/workflows/js-test.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      - uses: actions/checkout@v3	
+      - uses: actions/setup-node@v3	
+        with:	
+         node-version: '16.x'	
+         registry-url: 'https://registry.npmjs.org'	
+      - uses: actions/setup-java@v3	
+        with:	
+          java-version: '11'	
+          distribution: 'adopt'	
+      - name: Build AMS from source
```
Comment (mekya) [inline .github/workflows/js-test.yaml:21]: Building AMS from source is too much work. Why don't you get latest snapshot from website?

**Review by mekya — CHANGES_REQUESTED** [review]
Just take a look at my comment

File: .github/workflows/js-test.yaml (hunk 1 above)
Comment (mustafaboleken) [reply .github/workflows/js-test.yaml:21]: It could be much more efficient. Because of Usama is off for 2 weeks, I will refactor this pr. @mekya

**Comment by mustafaboleken** [comment]
@mekya just created [another pr](https://github.com/ant-media/StreamApp/pull/362) for the same purpose. I'm closing this right now.

**Comment by mekya** [comment]
Selenium tests are running enterprise side as java code. We have more control over the server-side settings to make tests more effective. 
We may think of moving them here. On the other hand,  it's not a priority for now. We'll revisit it later.

**Pull request closed without merging**